cinema/views.py

```diff
@@ -1 +1,65 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+from cinema.serializers import (
+    GenreSerializer,
+    ActorSerializer,
+    MovieSerializer, MovieSessionSerializer, CinemaHallSerializer,
+    MovieListSerializer, MovieRetrieveSerializer,
+    MovieSessionRetrieveSerializer,
+)
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
```

Comment:
Consider removing the `.all()` method before using `.prefetch_related()` or `.select_related()` to ensure efficient database queries. This is a best practice according to the task checklist.